nmi_mysql/nmi_mysql.py

```diff
@@ -134,6 +149,8 @@ def _execute(self, _query, _params, _multi=False):
                         'affected_rows': result.rowcount
                     }
 
+            con.close()
```

Comment:
Yes. Right now, we don't use transactions. I don't actually use transactions. But in the future, if we might need that, I think SQLAlchemy has some examples on using transactions.